moveit_ros/planning/planning_request_adapter_plugins/src/chomp_optimizer_adapter.cpp

```diff
@@ -164,21 +164,23 @@ class CHOMPOptimizerAdapter : public planning_request_adapter::PlanningRequestAd
     return "CHOMP Optimizer";
   }
 
-  virtual bool adaptAndPlan(const PlannerFn& planner, const planning_scene::PlanningSceneConstPtr& planning_scene,
+  virtual bool adaptAndPlan(const PlannerFn& planner, const planning_scene::PlanningSceneConstPtr& ps,
                             const planning_interface::MotionPlanRequest& req,
                             planning_interface::MotionPlanResponse& res,
                             std::vector<std::size_t>& added_path_index) const
   {
     // following call to planner() calls the OMPL planner and stores the trajectory inside the MotionPlanResponse res
     // variable which is then used by CHOMP for optimization of the computed trajectory
-    bool solved = planner(planning_scene, req, res);
+    bool solved = planner(ps, req, res);
```

Comment:
This could be `const` just as `planning_success`.

I'm also wondering if this function should return `solved && planning_success`?

Reply:
> This could be const just as planning_success.
???
> I'm also wondering if this function should return solved && planning_success?
Feel free to modify the return value.